fabric-model-loading-api-v1/src/client/java/net/fabricmc/fabric/api/client/model/loading/v1/BakedModelObserver.java

```diff
@@ -0,0 +1,26 @@
+package net.fabricmc.fabric.api.client.model.loading.v1;
+
+import net.minecraft.client.render.model.BakedModel;
+import net.minecraft.client.render.model.Baker;
+import net.minecraft.client.render.model.ModelBakeSettings;
+import net.minecraft.client.render.model.ModelLoader;
+import net.minecraft.client.render.model.UnbakedModel;
+import net.minecraft.client.texture.Sprite;
+import net.minecraft.client.util.SpriteIdentifier;
+import net.minecraft.util.Identifier;
+
+import java.util.function.Function;
+
+@FunctionalInterface
+public interface BakedModelObserver {
+	BakedModel observeBakedModel(Context context);
```

Comment:
The `BakedModel` should be moved to a parameter here. Not sure if other things should be moved too. (Perhaps the `Identifier`).

Reply:
Moved `BakedModel`, the other parameters are read-only so they should be fine staying inside the context.